.github/workflows/buildAndTest.yml

```
@@ -0,0 +1,78 @@
+name: Build & Test
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  scalafmt:
+    name: Check formatting
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache SBT ivy cache
+        uses: actions/cache@v1
+        with:
+          path: ~/.ivy2/cache
+          key: ${{ runner.os }}-sbt-ivy-cache-${{ hashFiles('**/build.sbt') }}
+      - name: Cache SBT
+        uses: actions/cache@v1
+        with:
+          path: ~/.sbt
+          key: ${{ runner.os }}-sbt-${{ hashFiles('**/build.sbt') }}
+      - name: Scalafmt
+        run: sbt scalafmtCheckAll
+
+  test:
+    name: Compile and Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache SBT ivy cache
+        uses: actions/cache@v1
+        with:
+          path: ~/.ivy2/cache
+          key: ${{ runner.os }}-sbt-ivy-cache-${{ hashFiles('**/build.sbt') }}
+      - name: Cache SBT
+        uses: actions/cache@v1
+        with:
+          path: ~/.sbt
+          key: ${{ runner.os }}-sbt-${{ hashFiles('**/build.sbt') }}
+      - name: Run tests
+        run: sbt test
```

Did we drop 2.12 already? I remember you said something about it. If not, we need to keep the `+` to compile for both versions:
```suggestion
        run: sbt +test
```

---

Support for Scala 2.12 will be dropped with #45 (which still yearns for your review :)).

---

In that case I would still add the `+` here and remove it in the other PR 😬 (I still have hopes to get cross-builds running)